packages/sitecore-jss/src/personalize/graphql-personalize-service.ts

```diff
@@ -11,6 +10,7 @@ export type GraphQLPersonalizeServiceConfig = CacheOptions & {
   timeout?: number;
   /**
    * Optional Sitecore Personalize scope identifier allowing you to isolate your personalization data between XM Cloud environments
+   * @deprecated Use 'scope' on 'PersonalizeMiddlewareConfig' instead. Will be removed in a future release.
```

Comment:
Had the same thought initially, but then decided it would be better to make the [personalize middleware config](https://github.com/Sitecore/jss/blob/dev/packages/create-sitecore-jss/src/templates/nextjs-xmcloud/src/lib/middleware/plugins/personalize.ts#L33) backwards compatible (which we can do by deprecating here, and then [using it as fallback in personalize middleware](https://github.com/Sitecore/jss/blob/8f11f0b2344fa916261956e1ee64b6ac53fcdc33/packages/sitecore-jss-nextjs/src/middleware/personalize-middleware.ts#L208)).

Good point about "PersonalizeMiddlewareConfig" though, I can remove that.